Device/ARMCA53/Include/ARMCA53.h

```diff
@@ -0,0 +1,180 @@
+/**************************************************************************//**
+ * @file     ARMCA53.h
+ * @brief    CMSIS-Core(A) Device Header File for Device ARMCA53
+ *
+ * @version  V1.0.0
+ * @date     18. July 2023
```

Comment:
Should this be the current date?

Reply:
should be the date of comit, right? --> fixed